docs/install/install.rst

```diff
@@ -0,0 +1,237 @@
+them directives and they start with :literal:`-D`. The defaults are marked with :strong:`bold` text.
+
+============================================ =================
+Directive                                    Description
+============================================ =================
+DFREECIV_ENABLE_TOOLS={:strong:`ON`/OFF}     Enables all the tools with one parameter (Ruledit, FCMP, 
+                                             Ruleup, and Manual)
+DFREECIV_ENABLE_SERVER={:strong:`ON`/OFF}    Enables the server. Should typically set to ON to be able 
+                                             to play AI games
+DFREECIV_ENABLE_NLS={:strong:`ON`/OFF}       Enables Native Language Support
+DFREECIV_ENABLE_CIVMANUAL={:strong:`ON`/OFF} Enables the Freeciv Manual application
+DFREECIV_ENABLE_CLIENT={:strong:`ON`/OFF}    Enables the Qt client. Should typically set to ON unless you 
+                                             only want the server
+DFREECIV_ENABLE_FCMP_CLI={ON/OFF}            Enables the command line version of the Freeciv21 Modpack 
+                                             Installer
+DFREECIV_ENABLE_FCMP_QT={ON/OFF}             Enables the Qt version of the Freeciv21 Modpack Installer 
+                                             (recommended)
+DFREECIV_ENABLE_RULEDIT={ON/OFF}             Enables the Ruleset Editor
+DFREECIV_ENABLE_RULEUP={ON/OFF}              Enables the Ruleset upgrade tool
+DCMAKE_BUILD_TYPE={:strong:`Release`/Debug}  Changes the Build Type. Most people will pick Release
+DCMAKE_INSTALL_PREFIX=/some/path             Allows an alternative install path. Default is 
+                                             :file:`/usr/local/share/freeciv21`
+============================================ =================
+
+For more information on other cmake directives see 
+https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html.
+
+Once the command line directives are determined, the appropriate command looks like this:
+
+.. code-block:: rst
+
+  $ cmake . -B build -G Ninja \
+     -DFREECIV_ENABLE_TOOLS=OFF \
+     -DFREECIV_ENABLE_SERVER=ON \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DFREECIV_ENABLE_NLS=OFF \
+     -DCMAKE_INSTALL_PREFIX=$HOME/Install/Freeciv21
+
+
+Compiling/Building
+******************
+
+Once the build files have been written, then compile with this command:
+
+.. code-block:: rst
+
+  $ cmake --build build
+
+
+Installing
+**********
+
+Once the compilation is complete, install the game with this command.
+
+.. code-block:: rst
+
+  $ cmake --build build --target install
+
+
+.. note:: If you did not change the default install prefix, you will need to elevate privileges 
+    with :file:`sudo`.
+
+After compilation, the important results are:
+
+  - The :file:`build/freeciv21-client` and :file:`build/freeciv21-server` binaries.
+  - The :file:`build/data/` directory, which contains the graphics and scenarios.
+

```

Comment:
Probably include a line saying where the desktop files are available ? and where it could be placed ? 
I could also say what each executable does.. atleast the `freeciv21-client`, `freeciv21-server` like a new game in client starts server in bg ? Or is that too much info.

Reply:
The `desktop` files aren't handled by the build system, see #402